recipes/pymt_cem/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.1.2" %}
+
+package:
+  name: pymt_cem
+  version: {{ version }}
+
+source:
+  url: https://github.com/pymt-lab/pymt_cem/archive/v{{ version }}.tar.gz
+  sha256: 18f6ffd1a138e286d8ba7e1480756195fcd40052441a0138fad7530194adf3f2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - cython
+    - numpy =1.11
```

Comment:
I think this should be just `- numpy` as per: https://conda-forge.org/docs/meta.html#building-against-numpy

Reply:
@scopatz Thanks. I'll make this change to `pymt_cem-feedstock` once it's rendered.

Reply:
Thanks!

Reply:
@ocefpaf, @scopatz This package shouldn't really have a minimum package requirement for *numpy*. I think I just want to use whatever the oldest version of numpy that's forwards compatible. conda-forge/pymt_cem-feedstock#2 fixes this.

Reply:
OK. In that case you can do as @scopatz said and leave it to our central pinning to add the oldest `numpy` supported in conda-forge. See https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L428-L430